src/pages/understand-stacks/stacking.md

```diff
@@ -35,6 +35,9 @@ If you would like to implement this flow in your own wallet, exchange, or any ot
 [@page-reference | inline]
 | /understand-stacks/integrate-stacking
 
+[@page-reference | inline]
```

Comment:
could we add a similar callout in the "integrate stacking" file? the CTA could be: try it with the CLI before you integrate it